README.md

```diff
@@ -118,8 +146,14 @@ compile 'com.trello.rxlifecycle2:rxlifecycle-components:2.0.1'
 // If you want to use Navi for providers
 compile 'com.trello.rxlifecycle2:rxlifecycle-navi:2.0.1'
 
+// If you want to use Android Lifecycle for providers
+compile 'com.trello.rxlifecycle2:rxlifecycle-android-lifecycle:2.0.1'
+
 // If you want to use Kotlin syntax
 compile 'com.trello.rxlifecycle2:rxlifecycle-kotlin:2.0.1'
+
+// If you want to use Kotlin syntax with Android Lifecycle
+compile 'com.trello.rxlifecycle2:rxlifecycle-android-kotlin:2.0.1'
```

Comment:
Speaking from previous experience, we should make a comment here that it's unreleased (or just remove this altogether). Otherwise people will start trying to use it before we release.